### DISCUSSION TITLE
When should I set "Snapshot" for?

### DISCUSSION THREAD

**Discussion by mandyohhh** [opensquare-network/collaboration#893] [Q&A]
When I try to submit a proposal, other than the traditional "Start/Ending Time", there is a selection field for "Snapshot". What does that mean and when should I set it for? The beginning of the voting or the end of the voting?

**Comment by wliyongfeng** [comment]
@mandyohhh OpenSquare off-chain voting is a solution for substrate based chains inspired by [snapshot.org](https://snapshot.org/#/) in the EVM ecosystem. It's an off-chain voting solution which means we can not lock users' voting balance like what will happen in on-chain governance process. So the solution is we need pick a hight for a proposal, and users' voting weight will be decided by their balances on this chose height.

> What does that mean and when should I set it for?

So it means your proposal's result will be calculated based on the voters' balances on the chosen height.

OpenSquare off-chain voting has following differences compared to snapshot:
1. Customized for substrate chains, like reading balance in the substrate chain's pallets instead of reading ERC20 contract.
2. It supports multiple chains existed assets. Taking [RMRK](https://voting.opensquare.io/space/rmrk-curation) as an example, we can use `RMRK` on different chains to vote.
3. Biased voting strategy support.
